app/src/components/market/sections/market_pooling/scalar_market_pool_liquidity.tsx

```diff
@@ -309,7 +309,7 @@ export const ScalarMarketPoolLiquidity = (props: Props) => {
 
       setTxState(TransactionStep.waitingConfirmation)
       setIsTransactionModalOpen(true)
-      let fundsAmount = formatBigNumber(depositedTokensTotal, collateral.decimals)
+      let fundsAmount = formatBigNumber(depositedTokensTotal, collateral.decimals, collateral.decimals)
```

Comment:
Third parameter for the formatBigNumber is how many decimals it should have. So you are putting 18 decimals.....I would suggest leaving it like before since if you don't provide third parameter it will use the default of 2 decimals which is needed in this case. Also then you don't need to use formatNumber in setMessage below.

Reply:
thanks, will remove the third param in formatBigNumber. I believe `formatNumber` is required for `setMessage` below because that's how it handles numbers < 0.01.

appreciate you reviewing this